### PR TITLE
Fixed SetupGuideForMacOS.md

### PR DESCRIPTION
**Pull request by lpqdao**

### CODE REVIEW

**Review by yuliumitre — APPROVED** [review]
The changes look good to me. Great work!

File: SetupGuideForMacOS.md [hunk 1]
```diff
@@ -164,16 +166,16 @@ Additionally, you must have credentials (api key) access to the **[Value Set Aut
 
 ### crd-request-generator configs
 
-1. cd `<drlsroot>/dtr` 
-2. In `webpack.config.dev.js`, set `https` to `false`. Your file should look something like this:
+1. cd `<drlsroot>/dtr/bin` 
+2. In `dev`, set `https` to `false`. Your file should look something like this:
```
Comment (KeeyanGhoreshi) [inline SetupGuideForMacOS.md:170]: this reminds me that https isn't supported by the app right now.  we should probably get that set up.  We can leave this step in here for now, I suppose.

File: SetupGuideForMacOS.md [hunk 2]
```diff
@@ -164,16 +166,16 @@ Additionally, you must have credentials (api key) access to the **[Value Set Aut
 
 ### crd-request-generator configs
```
Comment (KeeyanGhoreshi) [inline SetupGuideForMacOS.md:167]: This is the header for the DTR section, but it says request generator.  Not your change, but we may as well fix it here while we're at it.

Comment (KeeyanGhoreshi) [reply SetupGuideForMacOS.md:167]: Github won't let me comment further up, but`### dtr configs` above should be the request generator header.  They must've been accidentally swapped.

**Pull request merged**